src/main/java/org/fundacion/jala/converter/view/converter/SelectFile.java

```diff
@@ -71,6 +75,7 @@ public void actionPerformed(final ActionEvent e) {
 
     /**
      * Gets the Origin Path of File when selected.
+     *
```

Comment:
The return should be: a String  with fileOriginPath.

Reply:
it's already solved dani could you check it